distribution/lib/Standard/Table/0.0.0-dev/src/Grouping_Method.enso

```diff
@@ -0,0 +1,12 @@
+from Standard.Base import all
+
+polyglot java import org.enso.table.operations.AddGroupNumber
+
+## Specifies a method for grouping rows in `add_group_number`.
+type Grouping_Method
+    ## Group rows by the specified `group_by`.
+    Unique
+
+    ## Create the specified number of buckets with the same number of rows in
+       each bucket (except possibly the last one).
+    Equal_Count bucket_count:Integer
```

Comment:
@radeusgd Suggested we use the term 'bucket' here, such as `Equal_Size_Bucket`. I agree that 'bucket' might be a good word here, but since this is just one of several grouping methods, I'm not sure.

Reply:
I think this whole component should be talking about buckets instead of groups. I think group is too closely assocaited with aggregating,